tests/acceptance/bootstrap/SearchContext.php

```diff
@@ -122,6 +121,32 @@ public function userSearchesUsingWebDavAPI(
 			null,
 			$body
 		);
+	}
+
+	/**
+	 * @When user :user searches for :pattern using the WebDAV API
+	 * @When user :user searches for :pattern and limits the results to :limit items using the WebDAV API
+	 * @When user :user searches for :pattern using the WebDAV API requesting these properties:
+	 * @When user :user searches for :pattern and limits the results to :limit items using the WebDAV API requesting these properties:
+	 *
+	 * @param string $user
+	 * @param string $pattern
+	 * @param string|null $limit
+	 * @param TableNode|null $properties
+	 *
+	 * @return void
+	 * @throws Exception|GuzzleException
+	 */
+	public function userSearchesUsingWebDavAPI(
+		string    $user,
+		string    $pattern,
+		?string   $limit = null,
+		TableNode $properties = null
```

Comment:
```suggestion
		string $user,
		string $pattern,
		?string $limit = null,
		TableNode $properties = null
```

Reply:
Previously also `?` was not provided for `TableNode $properties = null`. So I did not keep it. Should I now?

Reply:
I don't think gaping is required, but anyway follow the standard.
Implicitly nullable parameter declarations is deprecated in PHP 8.4, so I think placing `?` would be good. It must fire a depreciation warning when not placing `?`. If not that's fine

Reply:
I don't see any deprecation warning while running the tests, for now.

Reply:
`$properties` is an optional arg. so it needs to be declared as optional. So here, it is good to go with

```php
?TableNode $properties = null
```

Reply:
> Previously also `?` was not provided for `TableNode $properties = null`. So I did not keep it. Should I now? And the gaping is done in that way to align with that of `TableNode $properties = null`

All things that we did previously might not be in good standards or could be just copy-paste issues. so please, follow the standards that we think are best currently.